solarforecastarbiter/io/reference_observations/pnnl.py

```diff
@@ -0,0 +1,168 @@
+"""Initialize site obs/forecasts and fetch/update obs for PNNL site."""
+
+import logging
+from pathlib import Path
+
+import pandas as pd
+
+from solarforecastarbiter.io.reference_observations import (
+    common, default_forecasts)
+
+logger = logging.getLogger('reference_data')
+
+DATA_PATH = Path('pnnl_data')
```

Comment:
pretty restrictive, maybe make it come from an environment variable?

Reply:
will you use an environment variable when you import data?

Reply:
yes. can still default to the current value